modules/services/sharing/src/main/kotlin/au/com/shiftyjelly/pocketcasts/sharing/SharingClient.kt

```diff
@@ -100,14 +101,18 @@ class SharingClient(
                 )
             }
             WhatsApp, Telegram, X, Tumblr, More -> {
-                Intent()
+                val intent = Intent()
                     .setAction(Intent.ACTION_SEND)
                     .setType("text/plain")
                     .putExtra(EXTRA_TEXT, data.sharingUrl(hostUrl))
                     .putExtra(EXTRA_TITLE, data.sharingTitle())
+                    .putExtra(EXTRA_SUBJECT, data.sharingSubject())
```

Comment:
Can you add tests for this property in `SharingClientTest`?

Reply:
b56beeb